web/base_viewer.js

```diff
@@ -830,9 +861,11 @@ class BaseViewer {
 
   forceRendering(currentlyVisiblePages) {
     let visiblePages = currentlyVisiblePages || this._getVisiblePages();
+    let scrollAhead = this._isHorizontallyScrolling ? this.scroll.right :
```

Comment:
`scrollDirection` would make me think it takes values indicating ‘up’, ‘down’, ‘left’, or ‘right’. `scrollAhead` is a boolean: true if the user is scrolling ‘ahead’ on the abstract ‘behind’/‘ahead’ axis (which aligns with the left/right axis if scrolling horizontally and the up/down axis otherwise), false otherwise. I used ‘ahead’ elsewhere in this PR to indicate the positive direction on this orientation-agnostic axis. Do you think something like ‘backward’/‘forward’ would be more clear?